nndt/datasets/miccai2015.py

```diff
@@ -0,0 +1,48 @@
+from nndt.datasets.dataset import dataset
+
+
+class MICCAI2015(dataset):
+    def __init__(self, name='left_adrenal_gland_10', to_path=None):
+        super().__init__()
+
+        __dict = {
```

Comment:
How our users can know about list of all available datasets? Is there any `.help()` or `list()` or `.dataset_list()` function?

Reply:
Added function .dataset_list() that returns the list of available subsets for each dataset. For MICCAI it's organs, for ACDC it's datasets of different sizes